pkg/api/unversioned/types.go

```diff
@@ -308,6 +308,13 @@ type APIVersions struct {
 	TypeMeta `json:",inline"`
 	// versions are the api versions that are available.
 	Versions []string `json:"versions"`
+	// a map of client CIDR to server address that is serving this group.
```

Comment:
To help clients reach servers in the most network-efficient way possible, a map of client CIDR to a server address believed reachable and ideal from that CIDR.